tests/conftest.py

```diff
@@ -14,3 +16,35 @@ def pytest_addoption(parser):
 @pytest.fixture
 def vcrmode(request):
     return request.config.getoption("vcrmode")
+
+
+if "PULP_LOGGING" in os.environ:
+
+    @pytest.fixture
+    def pulp_container_log(vcrmode):
+        if vcrmode == "live":
+            with subprocess.Popen(
+                [
+                    os.environ["PULP_LOGGING"],
+                    "logs",
+                    "-f",
+                    "--tail",
+                    "0",
+                    "pulp-ephemeral",
```

Comment:
Unsure of the exact mechanics here, but IIUC this will show all logs since the container started. Is that what you're looking for?

Reply:
Nah, the `--tail 0` part will only start capturing logs right now.